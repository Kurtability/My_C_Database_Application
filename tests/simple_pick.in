pick a 1
pick b 1
set a 1 2 3
pick a 3
pick a 4
set a 3
pick a 3
del a
pick a 1
bye
