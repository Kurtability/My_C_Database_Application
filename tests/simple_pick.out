> pick a 1 
no such key

> pick b 1
no such key

> set a 1 2 3 
ok

> pick a 3
3

> pick a 4
index out of range

> set a 3
ok

> pick a 3
index out of range

> del a 
ok

> pick a 1
no such key

> bye
bye
